docs/content/introduction/productvisie.md

```diff
@@ -66,38 +95,61 @@ Bij de start van dit traject hanteren we de volgende uitgangspunten:
   - Bij eventueel centraal aangeboden voorzieningen worden API's ontsloten via NLX
   - Grote informatiemodellen worden waar nuttig opgesplitst in kleinere informatiemodellen die hoog-frequent kunnen wijzigen
+  - De modernste bewezen techniek wordt toegepast, en de toegepaste techniek verandert mee met de ontwikkelingen door de jaren heen. We gaan over in een toestand die kan worden aangeduid als "permanent beta".
 
 - De [API en URI strategie](https://aandeslagmetdeomgevingswet.nl/digitaal-stelsel/documenten/documenten/api-uri-strategie/) zoals opgesteld binnen het programma Digitaal Stelsel Omgevingswet worden waar mogelijk toegepast.
 
+- Eisen rond [Duurzame toegankelijkheid](https://wiki.nationaalarchief.nl/pagina/DUTO:Kwaliteitseisen) worden vanaf het begin in de standaarden verwerkt.
+
+- Archivering speelt een rol bij de uitvoering van elk proces, vanaf de start tot aan [overbrenging](https://wiki.nationaalarchief.nl/pagina/DUTO:Overbrenging) of [vernietiging](https://wiki.nationaalarchief.nl/pagina/DUTO:Vernietigen). De gegevensstandaard voor archiefstukken [TMLO](https://archief2020.nl/nieuws/toepassingsprofiel-metadatering-lokale-overheden) is al [geïntegreerd in RGBZ en ImZTC](https://www.gemmaonline.nl/index.php/Metadateren_van_zaakdossiers_conform_het_TMLO).
+
 
 ## Realisatie
 
 De realisatie gaat van start zonder complete blauwdruk van wat gebouwd moet worden. Door user-stories zo goed mogelijk in te vullen en vanuit oogpunt architectuur in de gaten te houden dat designchoices worden gemaakt die ruimte openlaten voor verdere ontwikkeling in de richting van de visie, komt de nieuwe standaard stukje bij beetje tot stand.
 
-Ontwikkeling valt uiteen in 3 delen die niet los van elkaar gezien kunnen worden:
-- Opstellen en realiseren van een nieuwe ZDS 2.0 API
-- Beschikbaar maken van een Open Source referentieimplementatie
-- Realiseren van toepassingen voor burgers of gemeenten die gebruik maken van zaakgegevens uit het zakenregistratiecomponent, via de gestandaardiseerde API
-
 Om een snelle start te maken wordt gestart met alle gemeenten en partijen waarvan de interesse bekend is. Aanhakers zijn nadrukkelijk welkom.
 
-De ontwikkeling gebeurt geheel Open Source, in de GitHub repository [VNG-Realisatie/gemma-zaken](https://github.com/VNG-Realisatie/gemma-zaken).
+De ontwikkeling gebeurt in de GitHub repository [VNG-Realisatie/gemma-zaken](https://github.com/VNG-Realisatie/gemma-zaken).
 
-Ook de backlog wordt publiek bijgehouden, samengesteld uit GitHub issues op dit open [Kanban bord](https://github.com/VNG-Realisatie/gemma-zaken/projects/1).
+Ook de backlog wordt publiek bijgehouden, samengesteld uit GitHub issues op dit open [Scrum bord](https://github.com/VNG-Realisatie/gemma-zaken/projects/1).
 
-Bij de ontwikkeling van de API's wordt gestreefd naar backwards compatibility. 
+Bij de ontwikkeling van de API's wordt gestreefd naar backwards compatibility.
```

Comment:
Het is een beetje vaag "streven naar", maar ik zie niet goed in hoe we backwards compatible blijven als er nu SOAP services zijn gedefinieerd en we gaan naar een REST API. Ons uitgangspunt is RGBZ 2.0 maar dat is nog een concept dus ook niet echt iets om rekening mee te houden?

Wellicht aanpassen op welk vlak precies naar backwards compatibility wordt gestreefd, als dat er is. Zo wil ik wel proberen de naamgeving van attributen, objecttypen, etc. te behouden maar of dat echt "compatibiliteit" betekent....

Reply:
Ik zou dat willen bedoelen voor de doorontwikkeling van de nieuwe API en niet zozeer voor de oude StUF-koppelvlakken. Het laatste lijkt me onbegonnen en ook onbezonnen werk :-)

Hugo

Reply:
Voor Rdam houdt backwards compatibiliteit in dat we de nieuwe api specificatie kunnen gebruiken op ons huidige zaaksysteem.